app/images/thumbnails/thumb_polyfills.svg

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="400px" height="144px" viewBox="0 0 400 144" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 52.3 (67297) - http://www.bohemiancoding.com/sketch -->
+    <title>thumb_polyfills</title>
+    <desc>Created with Sketch.</desc>
+    <g id="thumb_polyfills" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Group-19" fill-rule="nonzero">
+            <g id="Group-18" transform="translate(0.000000, 114.000000)" stroke="#FFFFFF">
+                <rect id="Rectangle" fill="#FFFFFF" x="0.5" y="0.5" width="29" height="29" rx="14.5"></rect>
+                <rect id="Rectangle-Copy-19" stroke-width="2" x="39" y="1" width="218" height="28" rx="14"></rect>
+            </g>
+            <g id="Group-10" transform="translate(142.000000, 0.000000)" stroke="#FFFFFF">
+                <rect id="Rectangle-Copy-29" stroke-width="2" x="229" y="1" width="28" height="28" rx="14"></rect>
+                <rect id="Rectangle-Copy-30" fill="#FFFFFF" x="0.5" y="0.5" width="219" height="29" rx="14.5"></rect>
+            </g>
+            <g id="Group-13" transform="translate(95.000000, 38.000000)" stroke="#FFFFFF">
+                <rect id="Rectangle-Copy-25" fill="#FFFFFF" x="0.5" y="0.5" width="161" height="29" rx="14.5"></rect>
+                <rect id="Rectangle-Copy-27" stroke-width="2" x="171" y="1" width="86" height="28" rx="14"></rect>
+            </g>
+            <g id="Group-14" transform="translate(47.000000, 76.000000)">
+                <rect id="Rectangle-Copy-25" stroke="#FFFFFF" stroke-width="2" transform="translate(188.000000, 15.000000) scale(-1, 1) translate(-188.000000, -15.000000) " x="119" y="1" width="138" height="28" rx="14"></rect>
+                <rect id="Rectangle-Copy-27" fill="#FFFFFF" transform="translate(55.000000, 15.000000) scale(-1, 1) translate(-55.000000, -15.000000) " x="0" y="0" width="110" height="30" rx="15"></rect>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
Let's try to optimize the svg file so it's download size is smaller. We did that for the other svg files.

@keanulee Which tool did you use to compress svg files? Yuin can use the same tool here.

Reply:
I've been using the `gulp images` step in polymer-library-docs (which was removed in this repo because this usually one-off step takes a long time to do on every build). Will do the minification and push to this PR.

Reply:
👍